spec/facter/resolvers/aix/mountpoints_spec.rb

```diff
@@ -8,7 +8,10 @@
       '/opt' => { device: '/dev/hd10opt', filesystem: 'jfs2', options: ['rw', 'log=/dev/hd8'] },
       '/usr' => { available: '2.84 GiB', available_bytes: 3_049_021_440, capacity: '43.21%', device: '/dev/hd2',
                   filesystem: 'jfs2', options: ['rw', 'log=/dev/hd8'], size: '5.00 GiB', size_bytes: 5_368_709_120,
-                  used: '2.16 GiB', used_bytes: 2_319_687_680 } }
+                  used: '2.16 GiB', used_bytes: 2_319_687_680 },
+      '/var' => { available: '205.06 MiB', available_bytes: 215023616, capacity: '0.76%', device: '/dev/hd3', 
+                  filesystem: 'x', options: ['rw', 'nodev', 'log=/dev/hd3'], size: '206.64 MiB', size_bytes: 216678912, 
+                  used: '1.58 MiB', used_bytes: 1655296}}
```

Comment:
this makes rubocop fail on main, which is weird because it does not failed on PR:

```
Offenses:

spec/facter/resolvers/aix/mountpoints_spec.rb:12:61: C: Style/NumericLiterals: Use underscores(_) as thousands separator and separate every 3 digits with them.
      '/var' => { available: '205.06 MiB', available_bytes: 215023616, capacity: '0.76%', device: '/dev/hd3', 
                                                            ^^^^^^^^^
spec/facter/resolvers/aix/mountpoints_spec.rb:12:110: C: Layout/TrailingWhitespace: Trailing whitespace detected.
      '/var' => { available: '205.06 MiB', available_bytes: 215023616, capacity: '0.76%', device: '/dev/hd3', 
                                                                                                             ^
spec/facter/resolvers/aix/mountpoints_spec.rb:13:110: C: Style/NumericLiterals: Use underscores(_) as thousands separator and separate every 3 digits with them.
                  filesystem: 'x', options: ['rw', 'nodev', 'log=/dev/hd3'], size: '206.64 MiB', size_bytes: 216678912, 
                                                                                                             ^^^^^^^^^
spec/facter/resolvers/aix/mountpoints_spec.rb:13:120: C: Layout/TrailingWhitespace: Trailing whitespace detected.
                  filesystem: 'x', options: ['rw', 'nodev', 'log=/dev/hd3'], size: '206.64 MiB', size_bytes: 216678912, 
                                                                                                                       ^
spec/facter/resolvers/aix/mountpoints_spec.rb:14:49: C: Style/NumericLiterals: Use underscores(_) as thousands separator and separate every 3 digits with them.
                  used: '1.58 MiB', used_bytes: 1655296}}
                                                ^^^^^^^
spec/facter/resolvers/aix/mountpoints_spec.rb:14:56: C: Layout/SpaceInsideHashLiteralBraces: Space inside } missing.
                  used: '1.58 MiB', used_bytes: 1655296}}
                                                       ^
spec/facter/resolvers/aix/mountpoints_spec.rb:14:57: C: Layout/SpaceInsideHashLiteralBraces: Space inside } missing.
                  used: '1.58 MiB', used_bytes: 1655296}}
```

Reply:
@gimmyxd I saw this after it was merged and fixed it in this **PR** : https://github.com/puppetlabs/facter/pull/2419 on **commit** :https://github.com/puppetlabs/facter/pull/2419/commits/de9be7886ab842fdc48931f1d9e02367e30e01b8

Not sure why it passed in the PR tests